rome-core/src/main/java/com/rometools/rome/feed/rss/Enclosure.java

```diff
@@ -30,7 +30,7 @@
 public class Enclosure implements Cloneable, Serializable {
     private static final long serialVersionUID = 1L;
     private String url;
-    private long length;
+    private long length = -1L;
```

Comment:
Alternatively, to avoid the magic numbers, one could use `Long length` instead of `long length` and then later do `if(long != null)` instead of `if(long != 0)`. Just a thought

Reply:
In any other case I'd fully agree with you. Here, though, I'd probably still go with a primitive value as `-1` cannot be a valid value for _length_.

Generally speaking, though, `Long` very often is very useful :)

Reply:
Hi @markusherzog @neroux 

I was thinking about it and I agree with you. Please, review last commit to verify & validate the patch.

Thanks a lot again! Regards,
Antonio.

Reply:
My recommendation was actually the opposite ;) I would stick with `long`.